frontend/src/core/islands/bridge.ts

```diff
@@ -169,3 +171,11 @@ export class IslandsPyodideBridge implements RunRequests, EditRequests {
     });
   }
 }
+
+function makeRelativeWorkerUrl(url: string) {
+  return url.startsWith("./")
```

Comment:
Normally I test islands behind a proxy, and this isn't always consistent. Will try

Reply:
depends what i am testing - but mostly ill just run:
`pnpm dev:islands` and then access the server with `<script type="module" src="/src/core/islands/main.ts"></script>`